### PR TITLE
Addresses to-do's in #122

### LINKED ISSUE #122: Homepage style updates
Stub...

- Shadow effects on cards below search - these appear to be remnants of other style changes, and should be removed. 
- Background image in search. We can use this pre-launch window as an opportunity to finalize image selection and needed resolution here.

### PR DESCRIPTION
**Pull request by KimSarabia**
## Description

- [x] Fixes homepage facet styles to remove dramatic hover
- [x] Updates bg image for search bar

Misc updates

- [x] Removes hover effect for publication date sidebar
- [x] Removes conditional in homepage template that stops navbar from rendering on search error. It needs to stay visible so users can navigate back to the homepage.

Screen Shot 2022-06-13 at 4 23 18 PM

closes #122

### CODE REVIEW

File: themes/crl/templates/header.phtml [hunk 1]
```diff
@@ -2,15 +2,15 @@
 	<div class="navbar-header">
 		<a class="page-title" href="
 			<?=$this->url('home')?>">CRL Online Catalog </a>
-	</div> <?php if (!isset($this->layout()->renderingError)): ?> <div class="nav-area">
+	</div> <div class="nav-area">
```
Comment (ryan-jacobs) [inline themes/crl/templates/header.phtml]: Interesting find here. I guess the original template had some logic in here that was not compatible with some error cases.

File: themes/crl/less/home-facet.less [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Homepage facets styles.
 .searchHomeContent {
-  background-image: url("../images/search-bg.png");
+  background-image: url("../images/search-bg.webp");
```
Comment (ryan-jacobs) [inline themes/crl/less/home-facet.less]: I like this image. The only thing that feels a little strange about it is the blur. Was that blur added?

Also, webp has good cross-browser support, correct?

File: themes/crl/less/search-results.less [hunk 3]
```diff
@@ -114,7 +114,9 @@
       margin: 10px 0;
 
       .onlineaccess {
-        color: @crl-gray-dark;
+        padding: 5px;
+        color: @white;
+        background-color: @crl-gray-dark;
```
Comment (ryan-jacobs) [inline themes/crl/less/search-results.less]: Good to see this bold formatting. Note that I don't think this style applies globally, so it's not showing up on user-specific saved lists. I think we can move that part to #95 as a follow-up though.

**Review by ryan-jacobs — COMMENTED** [review]
This is looking good, my only actionable comments were on the image blur.

File: themes/crl/less/home-facet.less (hunk 2 above)
Comment (KimSarabia) [reply themes/crl/less/home-facet.less]: I did add the blur. The original image is a jpeg of low quality and the blur was to hide it. FWIW, it's very much inspired by Villanova's image, which also applied blur https://library.villanova.edu/

File: themes/crl/less/search-results.less (hunk 3 above)
Comment (KimSarabia) [reply themes/crl/less/search-results.less]: Will take a look

File: themes/crl/less/home-facet.less (hunk 2 above)
Comment (KimSarabia) [reply themes/crl/less/home-facet.less]: Added less blurry image per our conversation

**Pull request merged**